kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/PersistentVolumeClaimExample.java

```diff
@@ -39,7 +40,9 @@
   private static final Logger logger = LoggerFactory.getLogger(PersistentVolumeClaimExample.class);
 
   public static void main(String[] args) {
-    String master = "https://localhost:8443/";
+    String master = "https://localhost:8443";
+    String namespace = "default";
+    String storage_class = "my-local-storage";
```

Comment:
Please, use proper Java naming conventions for variables (i.e. `storageClass`)

Reply:
Should i use storageClassName instead  of storageClass because it is already in use in try block (line no 55).